07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,161 @@
+          </button>
+          <div class="collapse navbar-collapse justify-content-end" id="navbarNavDropdown">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Screenshots</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Team</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Pricing</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                  Drop Down
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Action</a></li>
+                  <li><a class="dropdown-item" href="#">Another action</a></li>
+                  <li><a class="dropdown-item" href="#">Something else here</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        </nav>
+    </section>
+    <section class="main-content text-center">
+        <h1 class="main-title">Simple Systeme Fordiscount</h1>
+        <p class="mt-3">Integer Curcus Bibendum Augue Ao Cursus</p>
+        <section class="app-detail">
+            <div class="row justify-content-center px-5">
+                <div class="col-sm-3">
+                    <img src="assets/img/cloud.svg" width="50" alt="">
+                    <p class="title">INTRODUCING WHATSAPP</p>
+                    <p class="content">Lorem ipsum is simply dummy test of the printing and typesetting industry</p>
+                    <a href="#">Read More</a>
+                </div>
+                <div class="col-sm-3">
+                    <img src="assets/img/planet.svg" width="50" alt="">
+                    <p class="title">USER FRIENDLY INTERFACE</p>
+                    <p class="content">Lorem ipsum is simply dummy test of the printing and typesetting industry</p>
+                    <a href="#">Read More</a>
+                </div>
+                <div class="col-sm-3">
+                    <img src="assets/img/asteroid.svg" width="50" alt="">
+                    <p class="title">BUILD THE APP EVERYONE LOVE</p>
+                    <p class="content">Lorem ipsum is simply dummy test of the printing and typesetting industry</p>
+                    <a href="#">Read More</a>
+                </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)